Gemfile.lock

```diff
@@ -1,78 +1,78 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (6.0.0)
-      actionview (= 6.0.0)
-      activesupport (= 6.0.0)
-      rack (~> 2.0)
+    actionpack (6.0.2.1)
+      actionview (= 6.0.2.1)
+      activesupport (= 6.0.2.1)
+      rack (~> 2.0, >= 2.0.8)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actionview (6.0.0)
-      activesupport (= 6.0.0)
+    actionview (6.0.2.1)
+      activesupport (= 6.0.2.1)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    activemodel (6.0.0)
-      activesupport (= 6.0.0)
-    activerecord (6.0.0)
-      activemodel (= 6.0.0)
-      activesupport (= 6.0.0)
-    activesupport (6.0.0)
+    activemodel (6.0.2.1)
+      activesupport (= 6.0.2.1)
+    activerecord (6.0.2.1)
+      activemodel (= 6.0.2.1)
+      activesupport (= 6.0.2.1)
+    activesupport (6.0.2.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
-      zeitwerk (~> 2.1, >= 2.1.8)
-    builder (3.2.3)
+      zeitwerk (~> 2.2)
+    builder (3.2.4)
     concurrent-ruby (1.1.5)
-    crass (1.0.4)
+    crass (1.0.5)
     double_entry (1.0.1)
       activerecord (>= 3.2.0)
       activesupport (>= 3.2.0)
       money (>= 6.0.0)
       railties (>= 3.2.0)
-    erubi (1.8.0)
-    i18n (1.6.0)
+    erubi (1.9.0)
+    i18n (1.7.0)
       concurrent-ruby (~> 1.0)
-    json (2.2.0)
+    json (2.3.0)
     jwt (2.1.0)
     jwt_signed_request (2.5.1)
       jwt (>= 1.5.0, < 2.2.0)
       rack
-    loofah (2.2.3)
+    loofah (2.4.0)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
     method_source (0.9.2)
     mini_portile2 (2.4.0)
-    minitest (5.11.3)
-    money (6.13.4)
+    minitest (5.13.0)
+    money (6.13.6)
       i18n (>= 0.6.4, <= 2)
-    nokogiri (1.10.4)
+    nokogiri (1.10.7)
       mini_portile2 (~> 2.4.0)
     pagerduty (2.1.2)
       json (>= 1.7.7)
-    rack (2.0.7)
+    rack (2.0.8)
```

Comment:
### [rack](https://rack.github.io/)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 2.0.7 → 2.0.8

[_~~change-log~~, [source-code](http://github.com/rack/rack)_]

:tada: Patched vulnerabilities:

- [CVE-2019-16782](https://nvd.nist.gov/vuln/detail/CVE-2019-16782)
  **Possible information leak / session hijack vulnerability**

  
  URL: https://github.com/rack/rack/security/advisories/GHSA-hrqr-hxpp-chr3

  There's a possible information leak / session hijack vulnerability in Rack.  Attackers may be able to find and hijack sessions by using timing attacks targeting the session id. Session ids are usually stored and indexed in a database that uses some kind of scheme for speeding up lookups of that session id. By carefully measuring the amount of time it takes to look up a session, an attacker may be able to find a valid session id and hijack the session.  The session id itself may be generated randomly, but the way the session is indexed by the backing store does not use a secure comparison.  Impact:  The session id stored in a cookie is the same id that is used when querying the backing session storage engine.  Most storage mechanisms (for example a database) use some sort of indexing in order to speed up the lookup of that id.  By carefully timing requests and session lookup failures, an attacker may be able to perform a timing attack to determine an existing session id and hijack that session.






<details>
<summary>Commits</summary>

A change of **14** commits. See the full changes on [the compare page](https://github.com/rack/rack/compare/2.0.7...2.0.8).

These are the first 10 commits:
- (83d4bd1) [try to ensure we always have some kind of object](https://github.com/rack/rack/commit/83d4bd12c7e88455d21230bc24ec3a543654e2aa)
- (77f3aab) [remove more nils](https://github.com/rack/rack/commit/77f3aab73089abe518f62c46268b104bacd7114b)
- (bb3d486) [use session id objects](https://github.com/rack/rack/commit/bb3d486644755b2e0c7824b3910db1a83c98fcd2)
- (2b205ed) [store hashed id, send public id](https://github.com/rack/rack/commit/2b205ed5a047d9e50a13bb7a411bc48745b515ec)
- (1c7e3b2) [remove || raise and get closer to master](https://github.com/rack/rack/commit/1c7e3b259f0741c869dcfbabeb3e0670c4d3f848)
- (4e32262) [remove NullSession](https://github.com/rack/rack/commit/4e322629e0c6698c75a3fb541a42571f8543c34c)
- (73a5f79) [revert conditionals to master](https://github.com/rack/rack/commit/73a5f79f6854eed81ecc3e5fb9f8154e967ccc49)
- (dc45a06) [Add the private id](https://github.com/rack/rack/commit/dc45a06b339c707c1f658c123ec7216151878f7a)
- (6a04bbf) [Fallback to the legacy id when the new id is not found](https://github.com/rack/rack/commit/6a04bbf6b742c305d3a56f9bd6242e6c943cc2ad)
- (3ba123d) [Also drop the session with the public id when destroying ses…](https://github.com/rack/rack/commit/3ba123d278f1085ba78fc000df954e507af2d622)


</details>